scripts/mods/thaumicwonders.zs

```diff
@@ -247,3 +238,174 @@ craft.remake(<thaumicwonders:creative_essentia_jar>, ['pretty',
+
+mods.thaumcraft.Crucible.removeRecipe('thaumicwonders:orderwool');
+mods.thaumcraft.Crucible.removeRecipe('thaumicwonders:orderglowstone');
+mods.thaumcraft.Crucible.removeRecipe('thaumicwonders:ordermagma');
+mods.thaumcraft.Crucible.removeRecipe('thaumicwonders:orderquartz');
+mods.thaumcraft.Crucible.removeRecipe('thaumicwonders:ordersandstone');
+mods.thaumcraft.Crucible.removeRecipe('thaumicwonders:orderprismarine');
+mods.thaumcraft.Crucible.removeRecipe('thaumicwonders:orderchorus');
+
+mods.thaumcraft.Crucible.removeRecipe('thaumicwonders:avariccoal');
+mods.thaumcraft.Crucible.removeRecipe('thaumicwonders:avaricdiamond');
+mods.thaumcraft.Crucible.removeRecipe('thaumicwonders:avaricemerald');
+mods.thaumcraft.Crucible.removeRecipe('thaumicwonders:avaricquartz');
+mods.thaumcraft.Crucible.removeRecipe('thaumicwonders:avaricamber');
+
+mods.thaumcraft.Crucible.removeRecipe('thaumicwonders:hedge_dragons_breath');
+
+mods.thaumicwonders.MeatyOrb.removeAll();
+for item in <ore:listAllmeatraw>.items {
+  if(!isNull(item)) mods.thaumicwonders.MeatyOrb.add(item, 1);
+}
+
+mods.thaumicwonders.CatalyzationChamber.removeAll();
+
+val clusterList = [
+  <jaopca:item_clusteraluminium>,
+  <jaopca:item_clusteramber>,
+  <jaopca:item_clusteramethyst>,
+  <jaopca:item_clusteranglesite>,
+  <jaopca:item_clusterapatite>,
+  <jaopca:item_clusteraquamarine>,
+  <jaopca:item_clusterardite>,
+  <jaopca:item_clusterastralstarmetal>,
+  <jaopca:item_clusterbenitoite>,
+  <jaopca:item_clusterboron>,
+  <jaopca:item_clustercertusquartz>,
+  <jaopca:item_clusterchargedcertusquartz>,
+  <jaopca:item_clustercoal>,
+  <jaopca:item_clustercobalt>,
+  <jaopca:item_clusterdiamond>,
+  <jaopca:item_clusterdilithium>,
+  <jaopca:item_clusterdimensionalshard>,
+  <jaopca:item_clusterdraconium>,
+  <jaopca:item_clusteremerald>,
+  <jaopca:item_clusteriridium>,
+  <jaopca:item_clusterlapis>,
+  <jaopca:item_clusterlithium>,
+  <jaopca:item_clustermagnesium>,
+  <jaopca:item_clustermalachite>,
+  <jaopca:item_clustermithril>,
+  <jaopca:item_clusternickel>,
+  <jaopca:item_clusterosmium>,
+  <jaopca:item_clusterperidot>,
+  <jaopca:item_clusterplatinum>,
+  <jaopca:item_clusterquartzblack>,
+  <jaopca:item_clusterredstone>,
+  <jaopca:item_clusterruby>,
+  <jaopca:item_clustersapphire>,
+  <jaopca:item_clustertanzanite>,
+  <jaopca:item_clusterthorium>,
+  <jaopca:item_clustertitanium>,
+  <jaopca:item_clustertopaz>,
+  <jaopca:item_clustertrinitite>,
+  <jaopca:item_clustertungsten>,
+  <jaopca:item_clusteruranium>,
+  <thaumcraft:cluster:1>,
+  <thaumcraft:cluster:2>,
+  <thaumcraft:cluster:3>,
+  <thaumcraft:cluster:4>,
+  <thaumcraft:cluster:5>,
+  <thaumcraft:cluster:6>,
+  <thaumcraft:cluster:7>,
+  <thaumcraft:cluster>
+] as IItemStack[];
+
+for itemCluster in clusterList {
+  for oreEntry in itemCluster.ores {
+    if (oreEntry.name.matches("^cluster.*")) {
+      print("Debug: ore name: " ~ oreEntry.name);
```

Comment:
I assume this debug code is temporary. If its not (sometimes its ok to list added/removed recipes) use `utils.log()` instead.